package.json

```diff
@@ -99,7 +99,7 @@
     "@hot-loader/react-dom": "^16.13",
     "@reach/auto-id": "^0.13.0",
     "@reach/combobox": "^0.12.1",
-    "@reach/menu-button": "0.7.4",
+    "@reach/menu-button": "0.8.6",
     "@reach/rect": "^0.16.0",
     "@reach/tabs": "^0.1.5",
     "@reach/tooltip": "^0.12.1",
```

Comment:
"Playing in xx seconds..." no longer centered

![image](https://user-images.githubusercontent.com/64950861/150263051-fffbdcef-fa1c-427b-bee5-830073d42da7.png)

There's also an open issue where this screen overflows in Mobile, but just fyi.  No need to fix in this PR specifically.

Reply:
![image](https://user-images.githubusercontent.com/94839278/150336184-d35260c0-b09c-4b48-8142-78e2d61c4621.png)

Reply:
I've also fixed the overflow on mobile. Not ideal, but better. I'm probably gonna replace all the links on this replay screen with buttons to make the available options more obvious.
![image](https://user-images.githubusercontent.com/94839278/150420624-4f28c1bf-4812-44b9-8db5-7b716550a8eb.png)